include/bluetooth/mesh/light_ctrl_srv.rst

```diff
@@ -0,0 +1,427 @@
+
+   Light Lightness Control Server illuminance regulator
+
+The illuminance regulator takes the state machine's target illuminance level as the reference level, and compares it to sensor data from an external illuminance sensor.
+The inputs are compared to establish an error for the regulator, which the regulator tries to minimize.
+The regulator contains a proportional (P) and an integral (I) component whose outputs are summarized to a light level output.
+
+The illuminance regulator's output level only comes into effect if the required output level is higher than the state machine's output light level.
+To get full benefit of the regulator, the state machine's light level output should generally be configured to a lower value than desired, while instead keeping the target illuminance levels high.
+This allows the regulator to conserve energy by taking advantage of the room's ambient lighting.
+
+The regulator operates in a compile time configurable interval between 10 and 100 ms.
+For each step, the regulator calculates the integral of the error since the last step, and adds this to an internal sum.
+This sum is multiplied by an integral coefficient, and summarized with the raw difference multiplied by a proportional coefficient.
+Both the internal integral sum and the output level is represented as an unsigned 16-bit integer.
+
+To reduce noise, the regulator has a configurable accuracy property, which allows it to ignore errors smaller than the configured accuracy (represented as a percentage of the light level).
+See :option:`CONFIG_BT_MESH_LIGHT_CTRL_SRV_REG_ACCURACY` and :cpp:enumerator:`BT_MESH_LIGHT_CTRL_PROP_REG_ACCURACY <bt_mesh_light_ctrl::BT_MESH_LIGHT_CTRL_PROP_REG_ACCURACY>` for more information.
+
+.. note::
+   The illuminance regulator implementation only supports integers in its configuration.
+   The fractional part of coefficients, accuracy, and target levels is ignored.
+
+Sensor input
+------------
+
+The regulator relies on regular sensor input data to function correctly.
+This sensor data must come from an external :ref:`bt_mesh_sensor_srv_readme` model, reporting the ambient light level with the :cpp:var:`bt_mesh_sensor_present_amb_light_level` sensor type.
+The Sensor Server should publish its sensor readings to an address the Light LC Server is subscribed to, using a common application key.
+
+The Light LC Server will process all incoming sensor messages, and use them in the next regulator step. The regulator depends on frequent readings from the sensor server to provide a stable output for the Lightness Server. If the sensor reports are too slow, the regulator might oscillate, as it attempts to compensate for outdated feedback. It is strongly recommended to use the Sensor :ref:`bt_mesh_sensor_publishing_delta` feature for this purpose, triggering frequent reports when the regulator is compensating for large errors, while keeping the mesh traffic low in stable periods.
```

Comment:
If something is strongly recommended, maybe move this to .. tip or .. note?

I'm not sure what "triggering frequent reports" part refers to. It looks like a consequence of using the feature, so maybe "thus triggering" or "for triggering"?

Also, once-sentence-per-line issues here.